spec/ad-services/eyeota/eyeota.spec.ts

```diff
@@ -95,7 +95,12 @@ describe('Eyeota', () => {
 	});
 
 	it('constructs proper url with context', async () => {
-		window.fandomContext = { site: null, page: null } as FandomContext;
+		window.fandomContext = {
+			site: null,
```

Comment:
```suggestion
			site: {},
```
and same with others, then you will not have to use `as FandomContext`